atom/browser/api/atom_api_session.cc

```diff
@@ -659,16 +659,17 @@ std::string Session::GetUserAgent() {
   return browser_context_->GetUserAgent();
 }
 
-void Session::GetBlobData(const std::string& uuid,
-                          const AtomBlobReader::CompletionCallback& callback) {
-  if (callback.is_null())
-    return;
+v8::Local<v8::Promise> Session::GetBlobData(const std::string& uuid) {
+  v8::Isolate* isolate = v8::Isolate::GetCurrent();
```

Comment:
While using `v8::Isolate::GetCurrent` is fine, the usual way to get isolate in a method is to put isolate in parameters.

```c++
v8::Local<v8::Promise> Session::GetBlobData(v8::Isolate* isolate, const std::string& uuid)
```